css-pseudo-4/Overview.bs

```diff
@@ -1147,6 +1147,20 @@ Tree-Abiding Pseudo-elements</h2>
   non-inheritable properties take their <a>initial values</a> as usual.
   [[CSS-CASCADE-4]]
 
+<h3 id=partlike>
+Part-Like Pseudo-Elements</h3>
+
```

Comment:
I think maybe we can inline the definition into the Tree-Abiding Pseudo-elements section, it feels a bit odd maybe as its own little section given that the next section is both Tree-Abiding and Part-Like?

Reply:
My only issue with that is that I'd want both names to be in the title. But if that's okay, then yeah, let's just fold them together.